### DISCUSSION TITLE
Missing configurtion tab under administration

### DISCUSSION THREAD

**Discussion by zakaria-ahmd20** [Security-Onion-Solutions/securityonion#12445] [General]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94662829/6d288291-2872-4dd1-b677-7a935ea5c590)
need to use suricata and missing config tab

**Comment by cm-ops** [comment]
That is an old 2.3 screenshot, TheHive was removed in `2.3.120`. If you want to see alerts from Suricata view `Alerts`. `Administration > configuration` is in 2.4. Recommend you upgrade to 2.4 - https://docs.securityonion.net/en/2.4/getting-started.html
